SQL/New_patches/2020-02-05-NoHostConfig.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM Config WHERE ConfigID IN (Select ID FROM ConfigSettings WHERE Name='url');
```

Comment:
i think this should be in cleanup not new patches

Reply:
done